src/src/com/microsoft/aad/adal/AuthenticationSettings.java

```diff
@@ -253,4 +255,31 @@ public void setReadTimeOut(int timeOutMillis) {
 
         this.mReadTimeOut = timeOutMillis;
     }
+
+    /**
+     * Method to enable/disable WebView hardware acceleration used in
+     * {@link AuthenticationActivity} and {@link AuthenticationDialog}. 
+     * By default hardware acceleration is enable in WebView.
+     *  
+     * @param enable
+     *             if true, WebView would be hardwareAccelerated else it 
+     *             would be disable.
+     *             
+     * @see #isWebViewHardwareAccelerated()
+     */
+    public void setWebViewHardwareAcceleration(boolean enable) {
```

Comment:
By default, hardware acceleration is always enable in View, which can be disabled. [ref: http://developer.android.com/guide/topics/graphics/hardware-accel.html ]. Due to hardware accelerated AuthenticationActivity WebView, we have seen couple of crashes on some devices so we want to disable it in our app.
Here, this new method can be used to disable webview hardware acceleration.  Although by renaming this method to disableWebViewHardwareAccelertaion, would also solve the purpose but for completeness purpose i.e. enable/disable and query hardware acceleration in a same session, I use this name, something similar to setSkipBorker.